dae/dae/import_tools/schema2_import_storage.py

```diff
@@ -91,23 +91,66 @@ def _do_load_in_impala(cls, project, hdfs_study_layout):
             partition_description=partition_description,
         )
 
-    def generate_import_task_graph(self) -> TaskGraph:
+    @classmethod
+    def _do_study_config(cls, project):
```

Comment:
Why is this a method of project and not the Schema2GenotypeStorage? Just more the method there and add as a last step in the task graph the call to this method.

Reply:
This is already in the storage. My bad.